### DISCUSSION TITLE
2024-04-14-musk

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#180] [General]
# 埃隆·马斯克传 - 读书笔记 - Tw93

Hi, I'm Tw93, a Frontend Engineer from China, also dabbling a bit in Swift, Rust, Python, and Sketch. I'm the creator of MiaoYan, Pake, Trendy Weekly, XRender, and WeexUi. Believing in long-termism, I am passionate about open source and enjoy experimenting with new ideas.Study最近利用早上的阅读时间陆陆续续把《埃隆·马斯克传》给看完了，很推荐工程师同学一读，虽然平时马斯克在 X 上表现有些奇怪，其实在书里其实是一个实打实的工程师，很擅长将技术和商业以及成本控制结合的一个牛人，不过也有不少缺点的一个人，随手记录一些文字用于备忘。同样看完这本书，作为特斯拉股东，愿意继续持有。

https://tw93.fun/2024-04-14/musk.html

**Comment by zxpsuper** [comment]
ss

**Comment by yexia553** [comment]
看起来是一本不错的书，mark

**Reply by tw93** [reply]
挺值得一读

**Comment by xiaoshame** [comment]
“我们要培养出一群像工程师一样思考的设计师，再培养出一群像设计师一样思考的工程师”

能做到这点真是太难了，佩服佩服

**Reply by tw93** [reply]
或者这也是工程师的发展之路，而非全部是代码

**Reply by yexia553** [reply]
但是国内的环境下，如果换工作，企业还是更重视硬技能

**Reply by xiaoshame** [reply]
很像经常说的具备产品思维

**Comment by bobetax-byte** [comment]
"保持快速迭代，勇于承担风险，行事风格粗暴，接受阶段性失败，再次发起尝试"，实质上在很多公司，这主张如何保证不被中下层扭曲？实质上工程师在做方案的时候，成本只是考虑之一，稳定性可能更重要？为了保证稳定性可用性，需要做很多的额外设计？

**Reply by tw93** [reply]
需要看公司的发展阶段，其实前期或者快速出效果的时期可以更加敏捷”粗暴“，但是在中后期或者上线后稳定性会很重要，包括稳定性也很取决于工程师本身的能力。

**Comment by super456** [comment]
正在看这本书，你写的书评很有深度👍🏻
